src/main/java/com/commercetools/sync/categories/helpers/CategoryReferenceResolver.java

```diff
@@ -96,27 +91,6 @@ public CategoryReferenceResolver(@Nonnull final CategorySyncOptions options,
             format(FAILED_TO_RESOLVE_CUSTOM_TYPE, draftBuilder.getKey()));
     }
 
-    /**
```

Comment:
We should not remove the reference resolution for parent category

Reply:
c1b406d4